packages/cli/src/cmds/preview/preview.ts

```diff
@@ -602,7 +606,7 @@ export default class Preview extends YargsCommand {
       undefined,
       {
         shell: true,
-        cwd: botRoot,
+        cwd: ngrokChecker.getNgrokBinFolder(),
```

Comment:
Please note that if we use
```typescript
{
  shell: true,
  cwd: botRoot,
  env: {
    PATH: `${ngrokChecker.getNgrokBinFolder()}${path.delimiter}${process.env.PATH ?? ""}`
  }
}
```
like that in extension, it does not work. `ngrok` will not be recognized as a shell command.

Reply:
Are we using `npx ngrok` or just `ngrok`? I suppose `Path` works for `npx`.

Reply:
We are using `npx ngrok`, and it does not work. I think it is because `PATH` will be injected into the ngrok process, but not used to resolve ngrok command.

Reply:
As we discussed offline, handle `PATH` when merging process env.